pkg/webhook/policy.go

```diff
@@ -193,6 +193,22 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 		return admission.Errored(http.StatusInternalServerError, err)
 	}
 
+	if *logStatsAdmission {
+		logging.LogStatsEntries(
+			h.opa,
+			log.WithValues(
```

Comment:
As in audit, just pass the appropriate pre-creates log object (you can add undefined values as necessary)

Reply:
actually, in this case, we don't have a precreated `log` in hand / as part of `h`. See lines like:

https://github.com/open-policy-agent/gatekeeper/blob/5466e5ea1f4c29d190f11651b8e608ad66f47471/pkg/webhook/policy.go#L253-L267

I'm game to refactor some of that but maybe not as part of this PR ?

Reply:
SGTM

Normally I'd say "just fix it, not that many lines", but the fact that audit and mutation are in the same package makes this harder.